### PR TITLE
Add URL of YouTube video in README.md

### PR DESCRIPTION
**Pull request by noorshaik95**

### CODE REVIEW

**Comment by haricharanbole** [comment]
Hey NoorUllah,

Here are some of my observations on your intro video. 

**Positives:**

- I've observed a good chronological structure to your introduction. You started with initial technology and tied the structure of your content with the technologies you've been working so far and neatly summed it up with your total JavaScript experience. 

- It's good that you have mentioned why you've moved to a given technology, particularly when you mentioned that you shifted your technology based on company's decision. It displays your flexibility for company needs.

- I liked the way you explained your challenging project.

**Areas of Improvement:**

- I would have loved to see more energy and enthusiasm. Given that you've good experience in JS, a little more enthusiasm would add great value to your delivery. 

- It would have been better if you had recorded the video in landscape mode. I feel it is more appropriate than portrait mode in formal scenarios. 

To summarize, you have good content and challenging experience. A little more enthusiasm will take your intro to next level. :)

Thanks,
Haricharan Bole

**Review by haricharanbole — COMMENTED** [review]
Hey NoorUllah,

Here are some of my observations on your intro video.

Positives:

I've observed a good chronological structure to your introduction. You started with initial technology and tied the structure of your content with the technologies you've been working so far and neatly summed it up with your total JavaScript experience.

It's good that you have mentioned why you've moved to a given technology, particularly when you mentioned that you shifted your technology based on company's decision. It displays your flexibility for company needs.

I liked the way you explained your challenging project.

Areas of Improvement:

I would have loved to see more energy and enthusiasm. Given that you've good experience in JS, a little more enthusiasm would add great value to your delivery.

It would have been better if you had recorded the video in landscape mode. I feel it is more appropriate than portrait mode in formal scenarios.

To summarize, you have good content and challenging experience. A little more enthusiasm will take your intro to next level. :)

Thanks,
Haricharan Bole
